doc/cprover-manual/quantifiers.md

```diff
@@ -0,0 +1,40 @@
+# Quantifiers
+
+### Definition
+
+`__CPROVER_forall{}` and `__CPROVER_exists{}` are both supported in function-contracts context.
+
+```
+__CPROVER_forall { *type identifier* : range; *expression* }
+__CPROVER_exists { *type identifier* : range; *expression* }
+```
+
+
+### Semantics
+
+These expressions support quantification in CBMC.  They have meaning both for
+bounded and unbounded verification by virtue of the range predicate, which
+defines minimum and maximum values for quantification.  These can be arbitrarily
+small and large, within the limits of the C type that is quantified over.
```

Comment:
I think this paragraph is missing a description of what forall and exists mean:
- `__CPROVER_forall { *identifier* *type*; *boolean expression* }`: all `*type*` values for `*identifier*` must satisfy `*boolean expression*`
- `__CPROVER_exists { *identifier* *type*; *boolean expression* }`: some (at least one) `*type*` value for `*identifier*` must satisfy `*boolean expression*`

Reply:
Thanks! Added it.